src/services/redirectToDestination.js

```diff
@@ -54,7 +56,18 @@ export function redirectToDestination(
       break;
     }
     case "report": {
-      redirectURL = import.meta.env.VITE_APP_STUDENT_QUIZ_REPORT_BASE_URL;
+      const abTestResult = await abTestService.getABTestData(
```

Comment:
So we're assuming in frontend that AB test is for reporting?

Maybe it should be general and `report` should be part of AB test JSON?